p4include/v1model.p4

```diff
@@ -133,14 +145,17 @@ parser Parser<H, M>(packet_in b,
                     inout standard_metadata_t standard_metadata);
 control VerifyChecksum<H, M>(in H hdr,
                              inout M meta);
+@pipeline
```

Comment:
The @pipeline annotation is needed because BMv2 JSON mandates only ingress and egress can be in the `pipeline` section. The original BMv2 backend hard-coded the name of ingress and egress in v1model when generating the `pipeline` section in JSON. When I wrote the visitor, I did not figure out a way to traverse only `ingress` and `egress`, and not `verifychecksum` and `updatechecksum` without hard-coding the names. They are all `Controlblock` from the visitor's point of view.